glib/README.md

```diff
@@ -0,0 +1,10 @@
+# glib
+
+This package provides the glib libraries
+
+## Usage
+
+Typically this is a runtime dependency that can be added to your
```

Comment:
surely glib is a build time dependency?

Reply:
In most cases I've seen, packages that build upon glib are linking to it, thus making it a runtime dependency.

Reply:
ugh, sorry. brain is in debian mode not habitat mode. You're right.

Reply:
Or maybe we're calling the same thing with different names?

Reply:
haha np :)

Reply:
haha np :)